storm/src/main/storm/mesos/logviewer/LogViewerController.java

```diff
@@ -96,7 +102,7 @@ protected ProcessBuilder createProcessBuilder() {
         Paths.get(System.getProperty("user.dir"), "/bin/storm").toString(),
         "logviewer",
         "-c",
-        "storm.log.dir=" + System.getenv("MESOS_SANDBOX") + "/logs",
+        "storm.log.dir=" + logdir,
```

Comment:
Ok. I spoke too soon. I see that you have done exactly what I wanted in the LogViewerController function above :)